src/graph.js

```diff
@@ -246,8 +247,12 @@ export default class Graph {
     return this._maximum(items) - this._minimum(items);
   }
 
+  _diff(items) {
+    return this._last(items) - this._first(items);
+  }
+
   _lastValue(items) {
-    if (this.aggregateFuncName === 'delta') {
+    if (this.aggregateFuncName === 'delta' || this.aggregateFuncName === 'diff') {
```

Comment:
```suggestion
    if (['delta', 'diff'].includes(this.aggregateFuncName)) {
```
For improved readability I'd suggest this small change, thanks for the contribution!